eng/WpfArcadeSdk/tools/ReferenceAssembly.props

```diff
@@ -11,7 +11,6 @@
 
   <!-- Categorize the repo location based on the hand crafted assembly location. -->
   <PropertyGroup>
-    <ReferenceProjectRepoLocation Condition="$(InternalHandCraftedReferenceProjects.Contains('$(NormalizedMSBuildProjectName);'))">Internal</ReferenceProjectRepoLocation>
     <ReferenceProjectRepoLocation Condition="$(ExternalHandCraftedReferenceProjects.Contains('$(NormalizedMSBuildProjectName);'))">External</ReferenceProjectRepoLocation>
```

Comment:
Where is this used?  Does this make sense anymore considering everything that has a ref assembly has moved?

Reply:
see ReferenceAssembly.targets

I think we still need this because we only want that target to run for reference assembly projects